packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -227,6 +227,11 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
             async () => this.getInitialGCSummaryDetails(),
         );
     }
+    createProps?: any;
+
+    public get pendingMessageCount() {
+        return this.containerRuntime.pendingMessageCount;
```

Comment:
I'd think this should be data store specific, i.e. filter count only to this data store.